wandb/sdk/wandb_config.py

```diff
@@ -122,11 +122,22 @@ def as_dict(self):
     def __getitem__(self, key):
         return self._items[key]
 
+    def _check_locked(self, key, ignore_locked=False) -> bool:
+        locked = self._locked.get(key)
+        if locked is not None:
+            locked_user = self._users_inv[locked]
+            if not ignore_locked:
+                wandb.termwarn(
+                    "Config item '%s' was locked by '%s' (ignored update)."
```

Comment:
Should we check if the key is locked by "sweep" here and show a specific message "You can't change this config parameter because it's part of the sweep. If you want to save a new copy you need to use a new key in config" as Carey suggested in the Jira ticket?

Reply:
i agree, the message is poor.   I mentioned it in future work, but let me see if I can do something simple.   I was trying not to couple config with sweeps but i think i could register a better error message when the locked keys are set.